core/activity/response.go

```diff
@@ -20,21 +20,24 @@
 package activity
 
 import (
-	"errors"
+	"fmt"
 	"net/url"
 	"path"
-	"strings"
 
 	"github.com/infracloudio/msbotbuilder-go/connector/client"
 	"github.com/infracloudio/msbotbuilder-go/schema"
+	"github.com/pkg/errors"
 )
 
 // Response provides functionalities to send activity to the connector service.
 type Response interface {
 	SendActivity(activity schema.Activity) error
 }
 
-const replyToAcitivityURL = "v3/conversations/{conversationId}/activities/{activityId}"
+const (
+	sendToConversationURL = "/v3/conversations/%s/activities"
```

Comment:
Please see how we can make version upgrades seemless. Eg. If we need to update SDK to v4 API.
Probably we can define constant  for MS Teams API version we  are using and change it globally whenever we are shifting API compatibility globally.